ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bfd.j2

```diff
@@ -23,6 +23,18 @@
 {%         set multiplier = router_bfd.multihop.multiplier | arista.avd.default('-') %}
 | {{ interval }} | {{ min_rx }} | {{ multiplier }} |
 {%     endif %}
+{%     if router_bfd.sbfd is arista.avd.defined %}
+
+### Router BFD SBFD Summary
+
+| Initiator Interval | Initiator Multiplier | Reflector Minimum RX | Reflector Local-Discriminator |
+| -------- | ---------- | ---------- | ---------- |
```

Comment:
It would be nice if the length of | --- | thingies matched the column title.

Reply:
Updated documentation alignment, thanks!